.changeset/afraid-crews-fetch.md

```diff
@@ -0,0 +1,4 @@
+---
+---
+
+chore: added typegen test for type with native asset
```

Comment:
Wait for merge of:

https://github.com/FuelLabs/fuels-ts/pull/3118

Reply:
🚢